packages/runtime/src/components/builtin/Text/Text.tsx

```diff
@@ -47,11 +48,21 @@ const Text = forwardRef(function Text(
   { id, text, width, margin }: Props,
   ref: Ref<BoxModelHandle & PropControllersHandle<Descriptors>>,
 ) {
+  const [element, setElement] = useState<HTMLElement | null>(null)
   const [editor, setEditor] = useState<Editor | null>(null)
   const [propControllers, setPropControllers] =
     useState<DescriptorsPropControllers<Descriptors> | null>(null)
   const controller = propControllers?.text
 
+  useEffect(() => {
+    if (element == null || controller == null) return
+
+    return pollBoxModel({
+      element,
```

Comment:
this has a red line when I attempt to poll the box model on the `editor`. need to find a way of accessing the dom node of the slate editor